TF_Service_dotNet/TouchFree/UpdateBehaviour.cs

```diff
@@ -18,13 +18,14 @@ public UpdateBehaviour(float framerate = TargetFPS)
         private void SetTimer(float framerate)
         {
             updateLoop = new(1000f / framerate);
-            updateLoop.AutoReset = true;
+            updateLoop.AutoReset = false;
             updateLoop.Enabled = true;
             updateLoop.Elapsed += OnTimerElapsed;
         }
 
         private void OnTimerElapsed(object sender, ElapsedEventArgs e)
         {
+            updateLoop.Start();
```

Comment:
Move the start to after the event invocation so there cannot be overlapping event invocations - it's currently a potential race condition.

Reply:
I swapped this to use a lock instead so that we wouldn't get overlapping invocations